birdhouse/optional-components/weaver/conf.extra-service.d/weaver.conf.template

```diff
@@ -0,0 +1,22 @@
+
+    # NOTE:
+    #   Redirect to internal network of twitcher with Weaver root endpoint and alias allows to set
+    #   the same 'magpie' permissions on the 'weaver' service defined by "WEAVER_MANAGER_NAME".
+    #   This allows verification of the same service user/group permissions references regardless
+    #   whether the *shortcut* Weaver endpoint, the alias or the explicit 'twitcher' proxy route is used.
+    location /${WEAVER_CONFIG} {
+        # redirect to secured EMS/ADES
+        return 302 https://${PAVICS_FQDN}${TWITCHER_PROTECTED_PATH}/${WEAVER_MANAGER_NAME};
+    }
+
+    location /${WEAVER_MANAGER_NAME} {
+        proxy_pass https://${PAVICS_FQDN}${TWITCHER_PROTECTED_PATH}/${WEAVER_MANAGER_NAME};
+        proxy_set_header Host $host;
+        proxy_buffering off;
+        include /etc/nginx/conf.d/cors.include;
+    }
+
+    # location where process job outputs will be accessible
+    location ^~ ${WEAVER_WPS_OUTPUTS_PATH}/ {
```

Comment:
Could we mount weaver under WPS_OUTPUTS_PATH like other static birds so that all outputs are together?